kratos/utilities/string_utilities.h

```diff
@@ -96,6 +96,13 @@ namespace StringUtilities
         const std::string& rString,
         const char Delimiter);
 
+    /**
+     * @brief This function replaces from a string all times a certain substring is repeated
+     * @param from The original string to be replaced
+     * @param to The string which replaces the substring
+     */
+    std::string KRATOS_API(KRATOS_CORE) ReplaceAllSubstrings(std::string str, const std::string& from, const std::string& to);
```

Comment:
```suggestion
    std::string KRATOS_API(KRATOS_CORE) ReplaceAllSubstrings(const std::string& rStringToBeReplaced, str, const std::string& rOldSubstring, const std::string& rNewSubstring);
```

seeing `std::string` being passed by value makes me cringe xD

also please name the arguments better, I cannot understand without looking at the code what is what (I only made a suggestion but pretty sure I got it wrong)

Reply:
Sorry, it was a function copy pasted from stackoverflow, one sec